concurrency/cmd/webhook/main.go

```diff
@@ -83,45 +83,21 @@ func newValidationAdmissionController(ctx context.Context, cmw configmap.Watcher
 	)
 }
 
-func newConcurrencyControlDefaultingAdmissionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
-	var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
-		concurrencyControlKind: &v1alpha1.ConcurrencyControl{},
-	}
-	return defaulting.NewAdmissionController(ctx,
-
-		// Name of the resource webhook.
-		"webhook.concurrency.custom.tekton.dev",
```

Comment:
Is this because the 2 webhooks have the same name i.e. this one and the the below pr defaulting one? Can we try renaming one of them to something else (with the corresponding new yaml for the new webhook)?

Reply:
I'm not sure what the reason for the error is. Dibyo and I chatted offline-- decided to merge as there isn't really a reason to prefer 2 webhooks